sdk/storage/azure-storage-common/inc/azure/storage/common/storage_common.hpp

```diff
@@ -64,7 +50,25 @@ namespace Azure { namespace Storage {
     /**
      * @brief The algorithm used for hash.
      */
-    HashAlgorithm Algorithm;
+    HashAlgorithm Algorithm = HashAlgorithm::Md5;
```

Comment:
I was thinking this question. Maybe it's not the best idea to define this value here.

Reply:
We should have an `HttpHeaders` struct in common package, because in all three services, the definition of this struct is the same. We can add a default constructor, where we init the default value of member `ContentHash` to `Md5`.

Reply:
This is not how .NET does it.
I'm wondering if it is for versioning purposes, to prevent breaking changes if one HttpHeaders needs to contain more than what it has at the moment.
We can check with .NET team or Ted or Jeffrey. However, I don't think we should make the change before we have this change approved.

Reply:
Talked offline, will remain as is for now.